cmd/run.go

```diff
@@ -147,9 +147,15 @@ func executeContinuousRun(ctx context.Context, executeSyncAtStartup bool, schedu
 
```

Comment:
Wouldn't it be better to check here if those are nil? If those are nil we should stop this goroutine completely I think.